File: docs/site/examples/verify.ts
import { Design } from '@dry/sdk';

// The Verify pane checks bounds; returning d keeps the canvas visible.
const d = new Design()
  .geometry(0.6, 0.2).extruder(true)
  .point(0, 0, 0.2).point(300, 0, 0.2);
d;

File: docs/site/examples/verify.py
import dry

# Verify against machine-safety contracts.
d = (dry.Design()
     .geometry(0.6, 0.2).extruder(True)
     .point(0, 0, 0.2).point(300, 0, 0.2))
report = d.verify("generic", bounds=[[0, 250], [0, 210], [0, 220]])
